components/authentication/validators.js

```diff
@@ -1,8 +1,10 @@
+import dotenv from 'dotenv'

```

Comment:
This probably doesn't need to be here because it's included and initialized in index.js